cub/cub/device/dispatch/dispatch_common.cuh

```diff
@@ -7,8 +7,6 @@
 
 #include <cuda/std/type_traits>
```

Comment:
This should also be after the include guard

Reply:
With the move of all utilities to advance iterators to a dedicated header file this particular include line is no longer necessary and had been removed.